### PR TITLE
UI Updates

### PR DESCRIPTION
**Pull request by zmarotrix**
Organized UI, changed a few names and renamed "No Starting Items" to "Random Starting Items" along with its tooltip. Still needs to be functionally updated. I'm going to work on that in a bit, but you can probably get it done a lot faster than I would so let me know if you do that.

### CODE REVIEW

File: Builder.cs [hunk 1]
```diff
@@ -369,7 +369,7 @@ private void WriteFreeItems(params Item[] items)
             foreach (var item in itemList)
             {
                 var startingItemValues = item.GetAttributes<StartingItemAttribute>();
-                if (!startingItemValues.Any() && !_settings.NoStartingItems)
+                if (!startingItemValues.Any() && !_settings.RandomStartingItems)
```
Comment (ZoeyZolotova) [inline Builder.cs]: Did you not bother removing the `!`?

File: Forms/MainForm.cs [hunk 2]
```diff
@@ -486,7 +486,7 @@ private void cHideClock_CheckedChanged(object sender, EventArgs e)
 
         private void cNoStartingItems_CheckedChanged(object sender, EventArgs e)
```
Comment (ZoeyZolotova) [inline Forms/MainForm.cs]: Method name needs to be renamed.

File: Forms/MainForm.cs [hunk 3]
```diff
@@ -381,7 +381,7 @@ private void cNutChest_CheckedChanged(object sender, EventArgs e)
 
         private void cStartingItems_CheckedChanged(object sender, EventArgs e)
```
Comment (ZoeyZolotova) [inline Forms/MainForm.cs]: Method name needs to be renamed.

File: Forms/MainForm.cs [hunk 4]
```diff
@@ -774,7 +774,7 @@ public void InitializeSettings()
             _settings.FixEponaSword = true;
             _settings.ShortenCutscenes = true;
             _settings.QuickTextEnabled = true;
-            _settings.UpdateChests = true;
+            _settings.UpdateChests = false;
```
Comment (ZoeyZolotova) [inline Forms/MainForm.cs]: Revert this. And don't try to sneak things in by omitting them from the pull request description.

File: Randomizer.cs [hunk 5]
```diff
@@ -1195,7 +1195,7 @@ private void PlaceFreeItems(List<Item> itemPool)
                 Item.StartingHeartContainer1,
                 Item.StartingHeartContainer2,
             };
-            var availableStartingItems = (_settings.NoStartingItems
+            var availableStartingItems = (_settings.RandomStartingItems
                 ? ItemUtils.AllRupees()
                 : ItemUtils.StartingItems())
```
Comment (ZoeyZolotova) [inline Randomizer.cs]: These are reversed, please fix them.

File: Forms/MainForm.cs (hunk 4 above)
Comment (zmarotrix) [reply Forms/MainForm.cs]: I was not trying to sneak anything in, I just forgot a small detail. This should not be on by default at all.

File: Randomizer.cs (hunk 5 above)
Comment (zmarotrix) [reply Randomizer.cs]: Reverse?

File: Forms/MainForm.cs [hunk 6]
```diff
@@ -87,7 +87,7 @@ private void InitializeTooltips()
             TooltipBuilder.SetTooltip(cFloors, "Select a floortype for every floor ingame:\n\n - Default: Vanilla floortypes.\n - Sand: Link sinks slowly into every floor, affecting movement speed.\n - Ice: Every floor is slippery.\n - Snow: Similar to sand. \n - Random: Any random floortypes of the above.");
             TooltipBuilder.SetTooltip(cClockSpeed, "Modify the speed of time.");
             TooltipBuilder.SetTooltip(cHideClock, "Clock UI will be hidden.");
-            TooltipBuilder.SetTooltip(cNoStartingItems, "You will not start with any randomized starting items.");
+            TooltipBuilder.SetTooltip(cRandomStartingItems, "You will start with any randomized items obtained during first cycle.\nSuch as Sword, Shield, Heart Containers, Song of Healing and Deku Mask.");
```
Comment (ZoeyZolotova) [inline Forms/MainForm.cs]: Wording is a bit weird. Sword, Shield, and Heart Containers aren't really "obtained" during first cycle.

File: Randomizer.cs (hunk 5 above)
Comment (ZoeyZolotova) [reply Randomizer.cs]: you reversed the `if` condition, so now you need to swap the `then` and the `else`

File: Forms/MainForm.cs (hunk 6 above)
Comment (zmarotrix) [reply Forms/MainForm.cs]: I added, "Obtained before or during first cycle". The list should also help clear that up.

File: Randomizer.cs (hunk 5 above)
Comment (zmarotrix) [reply Randomizer.cs]: Fixed.

File: Forms/MainForm.cs (hunk 3 above)
Comment (ZoeyZolotova) [reply Forms/MainForm.cs]: You still haven't renamed the method.

File: Forms/MainForm.cs (hunk 4 above)
Comment (ZoeyZolotova) [reply Forms/MainForm.cs]: Please revert this change as we discussed.

**Pull request merged**